src/utils/key.ts

```diff
@@ -25,3 +25,31 @@ export function getKey(key: string, dir?: "rtl" | "ltr"): string {
 }
 
 export const numberKeys = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+export const letterKeys = [
```

Comment:
I noted it here as well - https://github.com/Esri/calcite-components/issues/2127#issuecomment-839392889, but this doesn't match all browser's native behavior with native input - only Chrome limits this (and it allows the entry of a single "e"). 

I think it would be a reasonable feature, so long as we document that its a departure from native functionality - even though that native behavior is inconsistent (and maybe unexpected) across browsers! 

Thread on Bugzilla : https://bugzilla.mozilla.org/show_bug.cgi?id=1398528

Reply:
Didn't think about the Euler's constant, thanks for bringing that up!  I'll get some other folks' take on this.

Reply:
Cool - Just to be clear I’m not against having this baked in at all - most of the time it seems reasonable / expected -  so long as it’s documented / functionally doesn’t prevent folks from using the number input in a way they might expect a native one to!

Reply:
☝🏼 Agreed. We can add support for additional characters (e.g., `e`, `+`, `-`) in a follow-up PR.